include/ocpp/common/message_queue.hpp

```diff
@@ -498,20 +509,20 @@ template <typename M> class MessageQueue {
     }
 
     /// \brief pushes a new \p call message onto the message queue
-    template <class T> void push(Call<T> call) {
+    template <class T> void push(Call<T> call, const bool stall_until_accepted = false) {
```

Comment:
Maybe a bit of a nit but wouldn't true make more sense as the default argument since most messages need to be stalled?

Reply:
Only in the case the RegistrationStatus is Pending or Rejected messages will be pushed with `true` so most messages will likely be queued with `false`